data/transport missions.txt

```diff
@@ -1868,6 +1872,309 @@ mission "Terraforming 5"
+			choice
+				`	"Understood, officer."`
+					accept
+				`	"I'm wanted for what?"`
+				`	(Run away.)`
+					flee
+			
+			`	"The unlawful use of an asteroid, Captain. I'm not entirely sure what that means, but I've been told to send you to <planet>. Your compliance would be much appreciated."`
+			choice
+				`	"Alright. Hopefully I can get an answers as to what this is about."`
+					accept
+				`	(Run away.)`
+					flee
+			
+	on decline
+		"reputation: Republic" <?= -10
+	on fail
+		"reputation: Republic" <?= -10
+	on complete
+		conversation
+			`You are quickly escorted from the crowded <planet> spaceport to a court building in a wealthy district of the city and asked multiple questions pertaining to when you crashed an asteroid into Rand. "Why did you purposefully do crash an asteroid into a planet? What did you expect to happen? Do you know how many people could have died?" You answer the questions to the best of your ability, and after roughly an hour of questioning you are transferred to a different room where you find that Eric, Alaric, and Amy are all here as well.`
+			choice
+				`	"What are we going to do?"`
+					goto lawyer
+				`	"Do you know what we're doing here?"`
+			
+			`	"Apparently someone had a problem with us terraforming Rand," Amy says, looking rather disappointed.`
+			
+			label lawyer
+			`	"Don't worry," Alaric says. "Our lawyer is on his way right now. We'll all be out of here by the end of the day."`
+			`	Alaric's optimism turns out to be justified, as the lawyer quickly finds that there are no actual laws on the books that define lawful or unlawful uses of an asteroid. Since no one was hurt by the asteroid impact and data that Amy had been collecting shows that Rand's climate has changed for the better so far, the case is quickly dropped and you are escorted back to your ship.`
+			`	Before you are able to leave, Amy stops you. "Could you meet me in the spaceport bar, Captain? I have some good news to share with you."`
+
+
+
+mission "Terraforming 7"
+	name "Terraforming Research"
+	description "Bring Amy and Nolan to <destination> where they will research deeper into Amy's terraforming methods."
+	destination "Valhalla"
+	passengers 2
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Terraforming 6: done"
+	
+	on offer
+		conversation
+			`You find Amy in deep discussion with someone about her academy project, "Affordable Terraforming through Equilibrium Mapping." She takes a sip of her drink and invites you to sit down with them.`
+			`	"Hello, Captain! This is Nolan. He works for the Deep, and he contacted me after hearing about how we started terraforming Rand."`
```

Comment:
`works for the Deep`
Make some lore. University? Science organization? Government-sponsored conservation initiative? Feel free to touch up the earlier missions to build references and lore around his place of employment.

Disregard if this is expanded later and I just haven't gotten there yet.

`hearing about how we started terraforming Rand` Surely he didn't just 'hear' about the activities of some Dirt Belters who just happened to crash a freakin' asteroid onto their planet. That probably made the galactic news cycle for a month 😀